fiboa_cli/datasets/cz.py

```diff
@@ -0,0 +1,60 @@
+from ..convert_utils import convert as convert_
+import pandas as pd
+import numpy as np
+
+SOURCES = "https://mze.gov.cz/public/app/eagriapp/Files/geoprostor_zadosti23_2024-08-01_202409261243_epsg4258.zip"
+DATA_ACCESS = """
+"""
+
+ID = "cz"
+SHORT_NAME = "Czech"
+TITLE = "Field boundaries for Czech"
+DESCRIPTION = """The cropfields of Czech (Plodina)"""
+PROVIDERS = [
+    {
+        "name": "Czech Ministry of Agriculture (Ministr Zemědělství)",
+        "url": "https://mze.gov.cz/public/portal/mze/farmar/LPIS",
+        "roles": ["producer", "licensor"]
+    }
+]
+LICENSE = "CC-0"
+COLUMNS = {
+    'geometry': 'geometry',
+    'ZAKRES_ID': 'id',
+    'DPB_ID': 'block_id',
+    'PLODINA_ID': 'crop_code',
+    "PLOD_NAZE": "crop_name",
+    "ZAKRES_VYM": "area",
+    "DATUM_REP": "determination_datetime",
+    # 'OKRES_NAZE': 'administrative_area_level_2'  # Region - District
+}
+COLUMN_MIGRATIONS = {
+    'DATUM_REP': lambda col: pd.to_datetime(col, format="%d.%m.%Y")
+}
+
+MISSING_SCHEMAS = {
+    'properties': {
+        'crop_code': {
+            'type': 'string'
+        },
+        'crop_name': {
+            'type': 'string'
+        },
+        'block_id': {
+            'type': 'string'
+        },
+    }
+}
+
+def convert(output_file, cache = None, **kwargs):
+    convert_(
+        output_file, cache, SOURCES,
+        COLUMNS, ID, TITLE, DESCRIPTION,
+        license=LICENSE,
+        missing_schemas=MISSING_SCHEMAS,
+        column_migrations=COLUMN_MIGRATIONS,
+        providers=PROVIDERS,
+        explode_multipolygon=True,
```

Comment:
Can be removed, doesn't exist anymore, is applied automatically.

Reply:
Ah, that's it. The `explode_multipolygon` went from `_convert(**kwargs)` to a `gpd.read_file(..., **kwargs)` parameter and gave a console warning I couldn't explain.